lib/msal-browser/src/interaction_handler/InteractionHandler.ts

```diff
@@ -83,6 +84,47 @@ export abstract class InteractionHandler {
         return tokenResponse;
     }
 
+    async handleCodeResponseFromServer(authCodeResponse: AuthorizationCodePayload, state: string, authority: Authority, networkModule: INetworkModule): Promise<AuthenticationResult> {
```

Comment:
TODO: shares code with function above it, can probably be refactored with helper functions.